### PR TITLE
Adding support for the new attribute in the clusters view

### PR DESCRIPTION
**Pull request by joselsegura**
# Description

Adding version to the clusters endpoint

Fixes CCXDEV-8091

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing steps

N/A yet

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: server/api/v2/openapi.json [hunk 1]
```diff
@@ -1438,6 +1438,11 @@
                 "3": 0,
                 "4": 0
               }
+            },
+            "cluster_version": {
+              "type": "string",
+              "description": "Cluster version",
```
Comment (tisnik) [inline server/api/v2/openapi.json:1444]: Pls add "[Optional] Cluster version"

**Review by tisnik — APPROVED** [review]
LGTM

**Review by Bee-lee — APPROVED** [review]
Looks good

File: server/api/v2/openapi.json (hunk 1 above)
Comment (joselsegura) [reply server/api/v2/openapi.json]: Done

**Pull request merged**